.github/workflows/run_coverage.yml

```diff
@@ -54,9 +54,10 @@ jobs:
         run: |
           python -m pip install -e .  # must install in editable mode for coverage to find sources
           python -m pip list
+          python -m pip install --upgrade numpy
```

Comment:
Is this necessary? It is not part of the other workflows. If a higher version of numpy is required, then we should specify that elsewhere.

Reply:
I did specify a higher version in the requirements file but it didn't work. But I can try again given the updated workflows. Updating the numpy version was the fix for the codecov windows test.

Reply:
Yes, please try it again. If hdmf-zarr needs a higher numpy requirement than what is specified by the other packages in order to run, then it should be specified in `setup.py` and not only in this workflow.